theme/accessibility.less

```diff
@@ -14,9 +14,16 @@
 }
 
 @media not all and (pointer:coarse) {
-    *:not([tabindex='-1']):focus {
+    *:not([tabindex='-1']):focus,
+    [role='menuitem']:focus {
         outline: 2px solid #4D90FE !important;
     }
+
+    .blocklyWidgetDiv *:focus,
+    .blocklySVG *:focus,
+    .monaco-editor *:focus {
+        outline: 1px solid transparent !important;
```

Comment:
This won't use the default outline instead. You want to try and exclude these divs from your outline set above. Not the other way around.

Reply:
It's the first thing I tried but the ``:not`` selector doesn't let me exclude the children elements of a specific class, only the class itself.

Reply:
Then instead of using * but not, try just including the ones you want added. All semantic ui classes start with .ui 
that could be the solution